install/connections.csv

```diff
@@ -140,34 +140,34 @@ CSWB,10,White Bluffs,Portable,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CSWC,10,White Bluffs Solar Site with Comms,,,2013-07-24T05:05:00Z,2013-11-19T10:00:00Z
 CSWS,20,White Bluffs strong-motion,,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CTND,10,Courtenay,Portable,,2010-09-05T01:35:02Z,2010-09-29T00:00:00Z
-CTZ,10,Wharekauri,,,2007-12-15T07:00:00Z,2021-11-23T03:00:00Z
+CTZ,10,Wharekauri,,0,2007-12-15T07:00:00Z,2021-11-23T03:00:00Z
 CTZ,10,Wharekauri,Broadband,,2021-11-24T19:37:00Z,9999-01-01T00:00:00Z
-CTZ,20,Wharekauri,,,2007-12-15T07:00:00Z,2021-11-23T03:00:00Z
+CTZ,20,Wharekauri,,3,2007-12-15T07:00:00Z,2021-11-23T03:00:00Z
 CTZ,20,Wharekauri,Strong Motion,,2021-11-24T19:37:00Z,9999-01-01T00:00:00Z
 CVZ,10,Cave West Timaru,,,2014-05-16T00:00:15Z,9999-01-01T00:00:00Z
 DAGG,10,Dagg Sound,Portable,,2009-07-17T02:08:00Z,2009-09-14T21:34:00Z
 DCE,10,Deep Cove,Short Period,,1989-06-03T02:49:00Z,1989-06-09T18:00:00Z
 DCVF,10,Duck Cove,Short Period,,1993-03-22T04:51:00Z,1993-06-19T07:27:00Z
-DCZ,10,Deep Cove,,,2005-07-23T00:00:00Z,9999-01-01T00:00:00Z
-DCZ,20,Deep Cove,,,2005-07-23T00:00:00Z,2015-10-22T00:00:00Z
+DCZ,10,Deep Cove,,0,2005-07-23T00:00:00Z,9999-01-01T00:00:00Z
+DCZ,20,Deep Cove,,3,2005-07-23T00:00:00Z,2015-10-22T00:00:00Z
 DCZ,20,Deep Cove,Strong Motion,,2015-10-22T05:00:01Z,2021-08-09T21:08:00Z
-DIW,11,D'Urville Island,,,2003-12-10T00:00:00Z,2004-06-08T01:00:00Z
+DIW,11,Victoria University,EARSS Network,,2003-12-10T00:00:00Z,2004-06-08T01:00:00Z
```

Comment:
change to:
`DIW,11,D'Urville Island,EARSS Network,,2003-12-10T00:00:00Z,2004-06-08T01:00:00Z`

?

Reply:
Same deal, for DIW there was never a digitiser on the Island, only a sensor and an FM radio.